test/parallel/test-process-constrained-memory.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { Worker } = require('worker_threads');
+const constrainedMemory = process.constrainedMemory();
+if (constrainedMemory) {
```

Comment:
I assume `0` is not a valid return value.

```suggestion
if (constrainedMemory !== undefined) {
```

Reply:
Done !